styles/style.css

```diff
@@ -0,0 +1,242 @@
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format("woff2"),
+    url("../fonts/ptsans-400.woff") format("woff");
+  font-display: swap;
+}
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 400;
+  src: url("../fonts/ptsans-400.woff2") format("woff2"),
+    url("../fonts/ptsans-400.woff") format("woff");
+  font-display: swap;
+}
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/ptsans-700.woff2") format("woff2"),
+    url("../fonts/ptsans-700.woff") format("woff");
+  font-display: swap;
+}
+@font-face {
+  font-family: "PT sans";
+  font-style: normal;
+  font-weight: 700;
+  src: url("../fonts/ptsans-700.woff2") format("woff2"),
+    url("../fonts/ptsans-400.woff") format("woff");
+  font-display: swap;
+}
+
+body {
+  font-family: "PT sans";
+  font-size: 18px;
+  font-weight: normal;
+  font-style: normal;
+  line-height: 23px;
+  background-color: #ffffff;
+  color: #000000;
+}
+
+/*Pages*/
+
+.navigation-link {
+  font-family: PT Sans;
+  font-style: normal;
+  font-weight: bold;
+  font-size: 20px;
+  line-height: 26px;
+  color: #000000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+.main {
+  background-color: #e5e5e5;
```

Comment:
```suggestion
  background-color: #33464d;
```
Недостаточно контрастный
![image](https://user-images.githubusercontent.com/47776594/135767714-56f75480-0c7b-46b1-bfdc-8edfb196627d.png)